eng/pipelines/templates/jobs/mgmt-release.yml

```diff
@@ -0,0 +1,99 @@
+parameters: 
+- name: ShouldPublish
+  type: boolean
+  default: false
+- name: BuildToolsPath
+  type: string
+  default: '$(Build.SourcesDirectory)/azure-sdk-build-tools'
+- name: NugetVersion
+  type: string
+  default: '5.4.x'
+- name: PartnerDropsSubscription
```

Comment:
We don't need these extra configurations any longer since we are not publishing to blob.